src/tof/units.py

```diff
@@ -9,13 +9,8 @@ class Units:
     def __init__(self):
         self.alpha = const.m_n / const.h
 
-        self.Hz = sc.Unit('Hz')
-        self.s = sc.Unit('s')
-        self.us = sc.Unit('us')
-        self.angstrom = sc.Unit('angstrom')
-        self.m = sc.Unit('m')
-        self.deg = sc.Unit('deg')
-        self.rad = sc.Unit('rad')
+    def __call__(self, unit: str) -> sc.Unit:
+        return sc.Unit(unit)
```

Comment:
Wait, so you are using an instance of `Units` and its `__call__` operator to create other units? That looks very unusual.

More commonly, one would either use a constructor or a static method, such as `Units.from_string('m')`.

Reply:
I basically would like to have something that can both provide the specific `speed_to_wavelength` methods, as well as giving me units from a call.

Alternatively, I could just remove the class, have the `speed_to_wavelength` functions, and rely simply on Scipp units.